### PR TITLE
SAP deployment 8 February release

### PR DESCRIPTION
**Pull request by MarkvanMents**
Includes [MCDEP-401], [MCDEP-304], and [MCDEP-293]

### CODE REVIEW

File: content/releasenotes/developer-portal/sap-cloud-platform.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ These release notes cover changes to deployment to [SAP Cloud Platform](/develop
 
 For information on the current status of deployment to SAP Cloud Platform and any planned releases see [Mendix Status](https://status.mendix.com/).
 
+## 2021
+
+### February 8th, 2021
+
+* We added the ability to extend the timeout period for the Cloud Foundry health check when transporting an app to a new environment on SAP BTP.
+* We added the ability to enable *Dynatrace* for your app deployed to SAP BTP.
```
Comment (gauravkrshah) [inline content/releasenotes/developer-portal/sap-cloud-platform.md:18]: Added ability to define custom environment variables for you app deployed on SAP BTP. You can enable Dynatrace for your deployed to SAP BTP by selecting variables from supported list.

File: content/developerportal/deploy/sap-cloud-platform.md [hunk 2]
```diff
@@ -469,6 +484,20 @@ The services which are created by the Mendix Developer Portal will be named auto
 
 However, the maximum length for the service name is 50 characters. If this limit would be exceeded by the name created above, an alternative service name will be used. The format of this is **Environment Name** + **_** + **a random 6-character suffix**. If the Environment name is longer than 43 characters, only the first 43 characters are used.
 
+### 7.4 Runtime Tab{#runtime-tab}
+
+Use the Custom Environment Variables to **add**, **Edit**, or **Delete** an environment variable.
+
+The variables you add are chosen from a drop-down list.
+
+* **DT_PAAS_TOKEN** – the token for integrating your Dynatrace environment with Cloud Foundry
```
Comment (gauravkrshah) [inline content/developerportal/deploy/sap-cloud-platform.md:493]: this details is mentioned here https://github.com/mendix/cf-mendix-buildpack#dynatrace
Not sure we can prefer giving link to mention details here

**Review by gauravkrshah — CHANGES_REQUESTED** [review]
Kindly review comments.

File: content/developerportal/deploy/sap-cloud-platform.md (hunk 2 above)
Comment (mayankmendix) [reply content/developerportal/deploy/sap-cloud-platform.md:493]: Also, the change is of generic type, that is to add custom env variables. And for Dynatrace we have added DT variables in supported variables list.

File: content/developerportal/deploy/sap-cloud-platform.md [hunk 3]
```diff
@@ -270,21 +274,32 @@ When the package is ready to be deployed, a green tick will be shown next to the
...
 
 ## 7 Environment Details{#EnvironmentDetails}
 
-The environment details page contains three tabs: General, Model Options, and Services. Open the environment details by clicking **Details** on an environment on the Environments page of the Development Portal. You will also be taken to this page when you successfully deploy or transport your app.
+The environment details page contains the following four tabs:
+
+* [General](#general-tab) – how the application is deployed on SAP Cloud Platform
+* [Model Options](#model-options-tab) – application constants and scheduled events
+* [Services](#binding-services) – Cloud Foundry service management
+* [Runtime](#runtime-tab) – custom environment variables which control the behavior of the Mendix Runtime
```
[the remark this answers is not in the document]
Comment (MarkvanMents) [reply content/developerportal/deploy/sap-cloud-platform.md:292]: Should we include both uses of custom environment variables? They can only add the ones we specify, not any other variables they might want to give to the SAP Cloud Foundry environment?
```suggestion
* [Runtime](#runtime-tab) – custom environment variables which define "User-Provided Variables" in SAP cloud foundry environment which can be used to control the behavior of the Mendix Runtime
```

File: content/developerportal/deploy/sap-cloud-platform.md (hunk 2 above)
Comment (MarkvanMents) [reply content/developerportal/deploy/sap-cloud-platform.md:493]: I missed that in the UX - I'll add a bit more about "unsupported" variables.

File: content/releasenotes/developer-portal/deployment.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ Follow the links in the table below to see the release notes you want:
 | --- | --- |
 | [Mendix Cloud](mendix-cloud) | December 17th, 2020 |
 | [Mendix for Private Cloud](mendix-for-private-cloud) | January 6th, 2021 |
-| [SAP Cloud Platform](sap-cloud-platform) | December 21st, 2020 |
+| [SAP Cloud Platform](sap-cloud-platform) | February 8th, 2020 |
```
[the remark this answers is not in the document]
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/deployment.md:16]:
```suggestion
| [SAP Cloud Platform](sap-cloud-platform) | February 8th, 2021 |
```

File: content/developerportal/deploy/sap-cloud-platform.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MarkvanMents) [reply content/developerportal/deploy/sap-cloud-platform.md:292]: OK - I've now seen how you can add your own variables. I'll add this in the runtime tab section.

File: content/releasenotes/developer-portal/sap-cloud-platform.md (hunk 1 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/sap-cloud-platform.md:18]:
```suggestion
* We added the ability to define custom environment variables for you app deployed on SAP BTP.
* We added the ability to enable *Dynatrace* for your app deployed to SAP BTP by selecting variables from the supported custom environment variables list.
```

File: content/developerportal/deploy/sap-cloud-platform.md (hunk 2 above)
Comment (MarkvanMents) [reply content/developerportal/deploy/sap-cloud-platform.md:495]: I've added more information about the generic environment variables.
I think it is better to have a brief summary here than point people at the buildpack, which has lots of other information in it which may not be relevant.

**Comment by MarkvanMents** [comment]
SAP portal released.

**Pull request merged**